android/templates/snapshot/src/main/java/com/badoo/ribs/template/rib_with_view/foo_bar/FooBarNode.kt

```diff
@@ -0,0 +1,45 @@
+package com.badoo.ribs.template.rib_with_view.foo_bar
+
+import android.os.Bundle
+import android.view.ViewGroup
+import com.badoo.ribs.core.Node
+import com.badoo.ribs.template.rib_with_view.foo_bar.feature.FooBarFeature
+import io.reactivex.ObservableSource
+import io.reactivex.Single
+import io.reactivex.functions.Consumer
+
+class FooBarNode(
+    savedInstanceState: Bundle?,
+    viewFactory: ((ViewGroup) -> FooBarView?)?,
+    private val router: FooBarRouter,
+    private val input: ObservableSource<FooBar.Input>,
+    private val output: Consumer<FooBar.Output>,
+    private val feature: FooBarFeature,
+    interactor: FooBarInteractor // do NOT use interactor in workflows directly
```

Comment:
just a little bit strange that we are giving interactor field to the user, and asking not to use it.
Smells like design issue.

Reply:
Thanks. Fixed wording, hopefully it's better now.